lemur/common/utils.py

```diff
@@ -71,6 +71,23 @@ def parse_private_key(private_key):
     )
 
 
+def get_key_type_from_certificate(body):
+    """
+
+    Helper function to determine key type by pasrding given PEM certificate
+
+    :param body: PEM string
+    :return: Key type string
+    """
+    parsed_cert = parse_certificate(body)
+    if isinstance(parsed_cert.public_key(), rsa.RSAPublicKey):
+        return "RSA{key_size}".format(
+            key_size=parsed_cert.public_key().key_size
+        )
+    elif isinstance(parsed_cert.public_key(), ec.EllipticCurvePublicKey):
+        return get_key_type_from_ec_curve(parsed_cert.public_key().curve.name)
+
+
```

Comment:
Added this method to utils as it will be used by lemur application code in future

Reply:
unit testing here would be awesome, for RSA/ECC certs.
https://github.com/Netflix/lemur/blob/master/lemur/tests/test_utils.py

Reply:
Added [here](https://github.com/Netflix/lemur/pull/3150/files#diff-c09dceb332a94eea137a18f392af21aaR113-R116)